File: Q6.cpp
#include <iostream>

using namespace std;

int main()
{
    string nome;
    int numero_de_vezes, vezes = 1;
    cout << "Digite o seu nome: ";
    getline(cin, nome);
    cout << "Digite a quantidade de vezes que deseja que o seu nome apareça na tela: ";
    cin >> numero_de_vezes;
    for (int contador = 0; contador < numero_de_vezes; contador ++, vezes ++)
    {
        cout << vezes << "ª Vez: " << nome << "." << endl;
    }
    return 0;
}

File: Q6.cs
using System;

namespace Q6
{
    class Program
    {
        static void Main(string[] args)
        {
            string nome;
            byte número_de_vezes, vezes = 1;
            Console.Write("Digite o seu nome: ");
            nome = Console.ReadLine();
            Console.Write("Digite a quantidade de vezes que deseja que o seu nome apareça na tela: ");
            número_de_vezes = byte.Parse(Console.ReadLine());
            for (byte contador = 0; contador < número_de_vezes; contador ++, vezes ++)
            {
                Console.WriteLine("{0}ª Vez: {1}.", vezes, nome);
            }
        }
    }
}